GiEnJul/ClientApp/src/routes/registerGiver/Styles.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
this is the issue that causes the action to fail

Reply:
Thanks!!